package.json

```
@@ -35,10 +35,10 @@
     "prepublishOnly": "./scripts/prepare-artifacts.sh --network $npm_config_network"
   },
   "dependencies": {
-    "@keep-network/keep-core": ">1.8.0-dev <1.8.0-pre",
-    "@keep-network/tbtc": ">1.1.2-dev <1.1.2-pre",
+    "@keep-network/keep-core": ">1.8.0-dev <1.8.0-goerli",
+    "@keep-network/tbtc": ">1.1.2-dev <1.1.2-goerli",
```

Could we use the `development` tag here as well?

```suggestion
    "@keep-network/keep-core": "development",
    "@keep-network/tbtc": "development",
```